core/vm/logger.go

```diff
@@ -186,12 +185,12 @@ func (l *StructLogger) CaptureState(env *EVM, pc uint64, op OpCode, gas, cost ui
 			var (
 				address                   = common.Hash(stack.data[stack.len()-1].Bytes32())
 				internalContractAddr, err = contract.Address().InternalAddress()
-				value, _                  = env.StateDB.GetState(*internalContractAddr, address)
 			)
 			if err != nil {
-				log.Error("Error in CaptureState: " + err.Error())
+				fmt.Println(err.Error())
```

Comment:
we probably don't need format prints

Reply:
The logger uses fmt to print the logs

Reply:
but log has time and date information right and says Error before printing.

Reply:
Added "Error in CaptureState: " for context. But fmt must be used

Reply:
okay